dp-terraform/helm/rhacs-terraform/terraform_cluster.sh

```diff
@@ -157,7 +159,14 @@ invoke_helm "${SCRIPT_DIR}" rhacs-terraform \
   --set observability.observatorium.metricsClientId="${OBSERVABILITY_OBSERVATORIUM_METRICS_CLIENT_ID}" \
   --set observability.observatorium.metricsSecret="${OBSERVABILITY_OBSERVATORIUM_METRICS_SECRET}" \
   --set observability.pagerduty.key="${OBSERVABILITY_PAGERDUTY_ROUTING_KEY}" \
-  --set observability.deadMansSwitch.url="${OBSERVABILITY_DEAD_MANS_SWITCH_URL}"
+  --set observability.deadMansSwitch.url="${OBSERVABILITY_DEAD_MANS_SWITCH_URL}" \
+  --set vector.enabled=true \
```

Comment:
@0x656b694d let's set this to `false` - until AWS Terraform config is ready. So that we can merge this PR without breaking staging. And when AWS Terraform is ready and applied, we can just switch it on.
```suggestion
  --set vector.enabled=false \
```